core/src/main/java/org/apache/accumulo/core/fate/AbstractFateStore.java

```diff
@@ -62,16 +62,13 @@ public abstract class AbstractFateStore<T> implements FateStore<T> {
 
   public AbstractFateStore() {
     this.reserved = new HashSet<>();
-    this.defered = new HashMap<>();
+    this.deferred = new HashMap<>();
   }
 
   public static byte[] serialize(Object o) {
-    try {
-      ByteArrayOutputStream baos = new ByteArrayOutputStream();
-      ObjectOutputStream oos = new ObjectOutputStream(baos);
+    try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Same comment, this is also an issue in 2.1 https://github.com/apache/accumulo/blob/d97317945290fcd4e3a4eb8b3cf7e427fcc322bd/core/src/main/java/org/apache/accumulo/core/fate/ZooStore.java#L69